docs/decision_environment.rst

```diff
@@ -0,0 +1,36 @@
+====================
+Decision Environment
+====================
+
+Decision Environments are `Execution Environments <https://ansible-builder.readthedocs.io/en/stable/>`_ tailored towards running Ansible
+Rulebook tasks. These represent container images that launch and run the rulebook process and contain all of the dependencies, collections,
+and configuration needed to run a rulebook.
+
+A basic and minimal decision_environment is included at the root of the repository. This is a good starting point for building your own:
+
+.. code-block:: shell
+
+    ansible-builder build -f minimal-decision-environment.yaml -t minimal-decision-environment:latest
+
+This will build a container image named ``minimal-decision-environment:latest`` that can be used as the basis for your own decision environment.
+
+Lets run a rulebook using this decision environment, lets make sure we have a local inventory that has localhost in it we're going to use:
+
+.. code-block:: shell
+
+    echo "localhost ansible_connection=local" > inventory
+
+Now lets run a rulebook using this decision environment:
+
+.. code-block:: shell
+
+    docker run -it --rm -v ./inventory:/tmp/inventory ansible-execution-env:latest ansible-rulebook -r ansible.eda.hello_events -i /tmp/inventory
```

Comment:
This isn't going to work until this merges: https://github.com/ansible/event-driven-ansible/pull/131